Improve Spanish translations

pathros opened the pull request:
I reviewed all the Spanish translation files. Fixed minor misspelled words, improved translations in other cases.

pathros committed:
Spanish translation general check:
I reviewed all the Spanish translation files. Fixed minor misspelled words, improved translations in other cases.

pathros committed:
chore: fix code style

zepfietje commented on packages/support/resources/lang/es/actions/dissociate.php:
```diff
@@ -32,20 +32,20 @@
         'modal' => [
 
-            'heading' => 'Disociar :label seleccionada',
+            'heading' => 'Disociar :label seleccionadas',
```
@pathros, should this be `seleccionadas` or `seleccionados` to be consistent with the other translations?

pathros replied:
Yes, @zepfietje. You're right. Changed "seleccionadas" for "seleccionados" to be consistent.
Also, "disociada" and "disociadas" should be changed to "disociado" and "disociados" to be consistent.

zepfietje replied:
Perfect.

zepfietje commented on packages/tables/resources/lang/es/table.php:
```diff
@@ -17,11 +17,11 @@
 
         'bulk_select_page' => [
-            'label' => 'Selecionar/Deselecionar todos los elementos para las acciones masivas.',
+            'label' => 'Seleccionar/Deseleccionar todos los elementos para las acciones masivas.',
```
```suggestion
            'label' => 'Seleccionar/deseleccionar todos los elementos para las acciones masivas.',
```

pathros replied:
yes, you are right @zepfietje

zepfietje commented on packages/tables/resources/lang/es/table.php:
```diff
@@ -17,11 +17,11 @@
 
         'bulk_select_record' => [
-            'label' => 'Selecionar/Deselecionar :key para las acciones masivas.',
+            'label' => 'Seleccionar/Deseleccionar el elemento :key para las acciones masivas.',
```
```suggestion
            'label' => 'Seleccionar/deseleccionar el elemento :key para las acciones masivas.',
```

zepfietje commented on packages/tables/resources/lang/es/table.php:
```diff
@@ -35,7 +35,7 @@
         'label' => 'Navegación de paginación',
 
-        'overview' => 'Mostrando :first a :last de :total resultados',
+        'overview' => '{1} Se muestra un resultado|[2,*] Se muestran de :first a :last de un total de :total resultados',
```
Not sure if this string is too long for the UI?

pathros replied:
All right, we might leave it as the original one for simplicity sake. The original line is correct (in spelling and grammar), though.

zepfietje replied:
I think we should update it, because recently this translation was changed to read differently depending on the number of records. Maybe we should just leave out `de un total`?

The English version is `{1} Showing 1 result|[2,*] Showing :first to :last of :total results`.

pathros replied:
Yeah, indeed. I had originally updated it from the english version. It may seem long, but it depends on the Laravel pluralization. So, yes, let's leave it as `{1} Se muestra un resultado|[2,*] Se muestran de :first a :last de un total de :total resultados`

zepfietje replied:
Would `Se muestran de :first a :last de :total resultados` also be correct Spanish, @pathros?

pathros replied:
Yes, it is also correct. Either is fine. The only difference is the verbose stuff.

zepfietje reviewed (changes requested):
Thanks a lot for reviewing all Spanish translations, @pathros!
Just a couple comments to finish it up. 😊

zepfietje committed:
Update packages/tables/resources/lang/es/table.php

zepfietje committed:
Update packages/tables/resources/lang/es/table.php

pathros committed:
Update dissociate.php

Yes, @zepfietje. You're right. Changed "seleccionadas" for "seleccionados" to be consistent.

pathros committed:
Update table.php

The original line keeps it simple

pathros reviewed (commented):
Yes, all the suggestions you propose are correct!

Also, changed the "Disociada" for "Disociado" , "Disociadas" for "Disociados" in "[packages/support/resources/lang/es/actions/dissociate.php](https://github.com/filamentphp/filament/pull/6150#discussion_r1161087474)" file, for consistency sake.

pathros committed:
Merge branch 'filamentphp:2.x' into spanish-translations-check

pathros reviewed (commented):
All suggestions are correct, @zepfietje.

pathros committed:
Update table.php

Reverting to the original PR, to keep it consistent with the original English version.

pathros reviewed (commented):
Leaving as it is in the English version (but translated to Spanish)`{1} Se muestra un resultado|[2,*] Se muestran de :first a :last de un total de :total resultados`

pathros committed:
shorter version is also fine

`Se muestran de :first a :last de :total resultados` is correct. So we can leave it like that.

pathros reviewed (commented):
`Se muestran de :first a :last de :total resultados` is also correct. We can leave it like that.

zepfietje commented on packages/tables/resources/lang/es/table.php:
```diff
@@ -35,7 +35,7 @@
         'label' => 'Navegación de paginación',
 
-        'overview' => 'Mostrando :first a :last de :total resultados',
+        'overview' => 'Se muestran de :first a :last de :total resultados',
```
We should keep the different translations for 1 or many, since it will read really weird otherwise. I made it less verbose now. Could you do a final check if this is correct before I merge it, @pathros?

```suggestion
        'overview' => '{1} Se muestra un resultado|[2,*] Se muestran de :first a :last de :total resultados',
        
```

zepfietje committed:
Update packages/tables/resources/lang/es/table.php

zepfietje committed:
chore: fix code style
